frontend/public/components/monitoring/alertmanager/alertmanager-config.tsx

```diff
@@ -243,181 +252,324 @@ const deleteReceiver = (
-    confirmButtonVariant: ButtonVariant.danger,
-    onConfirm: () => deleteReceiver(secret, config, receiver.name, navigate),
-    ouiaId: 'AlertmanagerDeleteReceiverConfirmation',
-  });
-
-  const receiverMenuItems = (receiverName: string) => [
-    {
-      label: t('public~Edit Receiver'),
-      callback: () => {
-        const targetUrl = canUseEditForm
-          ? `/settings/cluster/alertmanagerconfig/receivers/${receiverName}/edit`
-          : `/settings/cluster/alertmanageryaml`;
-        return navigate(targetUrl);
-      },
-    },
-    {
-      label: t('public~Delete Receiver'),
-      isDisabled: !canDelete,
-      tooltip: !canDelete
-        ? t('public~Cannot delete the default receiver, or a receiver which has a sub-route')
-        : '',
-      callback: () => openDeleteReceiverConfirm(),
-    },
-  ];
-
-  return (
-    <>
-      <TableData className={tableColumnClasses[0]}>{receiver.name}</TableData>
-      <TableData className={tableColumnClasses[1]}>
-        {(receiver.name === InitialReceivers.Critical ||
-          receiver.name === InitialReceivers.Default) &&
-        !integrationTypesLabel ? (
-          <Link to={`/settings/cluster/alertmanagerconfig/receivers/${receiver.name}/edit`}>
-            {t('public~Configure')}
-            <PencilAltIcon className="co-icon-space-l" />
-          </Link>
-        ) : (
-          integrationTypesLabel
-        )}
-      </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        {isDefaultReceiver
-          ? t('public~All (default receiver)')
-          : _.map(receiverRoutingLabels, (rte, i) => {
-              return <RoutingLabels data={rte} key={i} />;
-            })}
-      </TableData>
-      <TableData className={tableColumnClasses[3]}>
-        <Kebab options={receiverMenuItems(receiver.name)} />
-      </TableData>
-    </>
-  );
+type ReceiverFilters = {
+  name: string;
 };
 
 interface ReceiversTableProps {
   secret: K8sResourceKind;
   config: AlertmanagerConfig;
   data: AlertmanagerReceiver[];
-  filterValue?: string;
 }
 
 const ReceiversTable: React.FC<ReceiversTableProps> = (props) => {
-  const { secret, config, filterValue } = props;
+  const { secret, config, data } = props;
   const { route } = config;
   const { receiver: defaultReceiverName, routes } = route;
   const { t } = useTranslation();
+  const label = t('public~Receivers');
+  const navigate = useNavigate();
 
   const routingLabelsByReceivers = React.useMemo(
     () => (_.isEmpty(routes) ? [] : getRoutingLabelsByReceivers(routes)),
     [routes],
   );
 
-  const EmptyMsg = () => (
-    <ConsoleEmptyState
-      title={t('public~No Receivers match filter {{filterValue}}', { filterValue })}
-    />
+  const openDeleteReceiverConfirm = useWarningModal();
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const { filters, onSetFilters, clearAllFilters } = useDataViewFilters<ReceiverFilters>({
+    initialFilters: { name: '' },
```

Comment:
Shall we omit the filter option dropdown since only one option(by `Name` ) is supported? 
<img width="1333" height="687" alt="alertmanager-config" src="https://github.com/user-attachments/assets/364d2adb-e8e5-4148-855c-1b3e461e2c63" />

for comparison,
<img width="1469" height="713" alt="Screenshot 2025-10-22 at 11 53 17 AM" src="https://github.com/user-attachments/assets/82259b35-01da-4677-978b-1f76bb6752f0" />